pkg/actions/nodegroup/create.go

```diff
@@ -289,7 +290,7 @@ func checkARMSupport(ctl *eks.ClusterProvider, clientSet kubernetes.Interface, c
 		if err != nil {
 			return err
 		}
-		if !upToDate {
+		if checkOutdatedAddons && !upToDate {
 			logger.Critical("to create an ARM nodegroup kube-proxy, coredns and aws-node addons should be up to date. " +
 				"Please use `eksctl utils update-coredns`, `eksctl utils update-kube-proxy` and `eksctl utils update-aws-node` before proceeding.")
 			return errors.New("expected default addons up to date")
```

Comment:
worth keeping some sort of log, just not erroring when it occurs. WDYT?

Reply:
Hmmmm.. what information do you think we should log and in which case? Do you mean leaving the current log in both cases, just changing the return behavior?

Reply:
yeah leave the current log, change the return behaviour. Although thinking over it again if your providing this flag you probably already know and don't care... so perhaps its not worth it

Its probably worth updating the log line to explain that you can get around this check by providing the skip flag though :smile:

Reply:
Good idea! I have updated the log =D